code-challenges.js

```diff
@@ -6,9 +6,14 @@ var collections = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example1 (can be a different order): ["yellow", "blue", "pink", "green"]
 // Expected output example2 (can be a different order): ["blue", "green", "pink"]
 // Expected output example3 (can be a different order): ["pink", "green"]
-
-
-
+filterArr = []
+const shuffle = (array) => {
+  let filterArr = array.map(value => {
+      return array[0] += 1
+  })
+  return filterArr.shift()
+}
+console.log(shuffle(collections))
 
```

Comment:
When I run this code this is the outcome I get:
```
purple1
purple111111
purple11111111111
```
Can you help me understand what your process is here?